cmd/root.go

```diff
@@ -264,3 +269,13 @@ func readStdin() (string, error) {
 	returnString := string(output)
 	return returnString, nil
 }
+
+// Switch between table and csv dependent on num columns
+func shouldUseTable(rows *sql.Rows) bool {
+	columns, err := rows.Columns()
+	if err != nil {
+		fmt.Println(err)
+		return true //default to using a table
+	}
+	return len(columns) <= 5
+}
```

Comment:
I think this function is small enough to be in-lined within the `default` case of the `switch` above, just to have one less step of indirection for now. Also, please remove the `fmt.Println(err)` in the error handler there.

The underlying logic makes sense to me though - I wonder if the `<= 5` can be made smarter in the future by inspecting the width of the first N lines and comparing to the terminal width. Maybe worth adding a comment or TODO there?